monix-reactive/shared/src/main/scala/monix/reactive/observables/CombineObservable.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2014-2018 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.observables
+
+import cats.Apply
+import monix.execution.internal.Newtype1
+import monix.reactive.Observable
+
+/** Newtype encoding for an [[Observable]] datatype that has a [[cats.Apply]]
+  * instance which uses [[Observable.combineLatest]] to combine elements
+  * needed for implementing [[cats.NonEmptyParallel]]
+  */
+
```

Comment:
Please delete this newline here. Personal pet peeve, plus Scaladoc might malfunction.